app/views/admin/_admin_nav.html.erb

```diff
@@ -29,7 +29,7 @@
     </li>
   <% end %>
   <li>
-    <%= span_if_current ts("Known Issues", key: "header"), known_issues_path %>
+    <%= span_if_current t(".known_issues"), known_issues_path %>
```

Comment:
This is missing the `<% if policy(KnownIssue).admin_index? %>` check, it got removed in the merge.

Reply:
Thanks for catching that!